2-classes-db/task.md

```diff
@@ -1,75 +1,112 @@
 # Class Database
+
 Run the following commands:
+
 ```
 createdb homework
 psql -d homework -f setup.sql
 ```
+
 This will create and populate a database called homework.
 
 It is based on real data from the government found at
 https://data.gov.uk/dataset/72eaec8e-0d32-4041-a553-87b852abee64/spend-over-25-000-in-worcestershire-acute-hospitals-nhs-trust
 
 You can work with it by running
+
 ```
 psql homework
 ```
+
 ## Submission
 
 Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands after each question.
 
 ### 1. Show the date, transaction_no, description and amount for those transactions whose amount is between £30,000 and £31,000.
+
 ```sql
+ANSWER: SELECT date, transaction_no, description, amount FROM spends WHERE amount >= 30000 AND amount <= 31000;
+
+SELECT date, transaction_no, description, amount FROM spends WHERE amount BETWEEN 30000 AND 31000;
 
 ```
+
 ### 2. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee'.
+
 ```sql
+ANSWER: SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE description LIKE '%fee%';
 
 ```
+
 ### 3. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'Fee'.
+
 ```sql
+SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE description LIKE '%Fee%';
 
 ```
+
 ### 4. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee' (case insensitive). You will need to search 'https://www.postgresql.org/docs/' to solve this.
+
 ```sql
+ANSWER: SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE lower(description) LIKE '%fee%';
 
 ```
+
 ### 5. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose amount is £25,000, £30,000, £35,000 or £40,000.
+
 ```sql
+ANSWER: SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE amount = 25000 OR amount = 30000 OR amount = 35000 OR amount = 40000;
 
```

Comment:
CORRECT - Do note, however, that you could have simplified the WHERE clause to `amount IN (25000,30000,35000,40000)`